src/generic/MPoly.jl

```diff
@@ -66,7 +66,7 @@ ring.
 """
 function gens(a::MPolyRing{T}) where {T <: RingElement}
    n = a.num_vars
-   return [gen(a, i, Val{a.ord}) for i in 1:n]
+   return elem_type(a)[gen(a, i, Val{a.ord})::elem_type(a) for i in 1:n]
```

Comment:
```suggestion
   return elem_type(a)[gen(a, i, Val(a.ord))::elem_type(a) for i in 1:n]
```
`Val` objects are supposed to be created with `()`, the `{}` is their type

Reply:
Yes, but the methods expect their type. It is unstable anyway.

Reply:
Ok. I don't really care.

Reply:
`Val{a.ord}` and `Val(a.ord)` do different things and unless the suggestion was to rewrite everyhing, I don't understand the comment.

Reply:
please just ignore the comment